lib/plugins/create/create.js

```diff
@@ -17,6 +17,7 @@ const validTemplates = [
   'aws-java-maven',
   'aws-java-gradle',
   'aws-kotlin-jvm-maven',
+  'aws-kotlin-node',
```

Comment:
Should we rename this to `aws-kotlin-node-gradle`? (since `gradle` is used as a build tool?)

Reply:
Yep, consistent with `aws-kotlin-jvm-maven`